packages/react-components/src/templates/NetworkPageHeader.tsx

```diff
@@ -16,9 +18,27 @@ const containerStyles = css({
   }em `,
 });
 
+const textStyles = css({
+  maxWidth: `${610 / perRem}em`,
+});
+
 const controlsStyles = css({
```

Comment:
This IMO breaks way too soon (e.g. 768px width, small desktop rotated). Not sure what design specs say about this